lib/task-data/schemas/dell-racadm-control.json

```diff
@@ -18,6 +18,18 @@
             "description": "Specify if reboot after racadm control tasks are done",
             "type": "boolean"
         },
+        "ip": {
```

Comment:
Same question: why those schema is not added to properties? And support you should only need one.